examples/aes-rs/Makefile

```diff
@@ -26,10 +26,10 @@ TARGET_HOST ?= $(TARGET)
 TARGET_TA ?= $(TARGET)
```

Comment:
This default assignment can be dropped with the renaming in this patch-set?

Reply:
Considering `examples/*/host/Makefile` and `examples/*/ta/Makefile` already have default assignment, I think this Makefile can be simplified to "
all:
	@make -s -C host
	@make -s -C ta

clean:
	@make -s -C host clean
	@make -s -C ta clean
", any comments?

Reply:
I am sceptical if that would allow us to override the defaults simply via `make TARGET_TA=<>` assignments but rather require explicit `export TARGET_TA=<>`. Please double check.

Reply:
Considering `make TARGET_TA=<>` it's better to keep those variables. Have updated to 
```
CROSS_COMPILE_HOST ?= aarch64-linux-gnu-
CROSS_COMPILE_TA ?= aarch64-linux-gnu-
TARGET_HOST ?= aarch64-unknown-linux-gnu
TARGET_TA ?= aarch64-unknown-linux-gnu
```